typescript/src/start-script.ts

```diff
@@ -139,6 +138,5 @@ try {
       });
     });
 } catch (err) {
-  console.error(err.toString());
-  process.exit(1);
+  throw new Error(err.toString());
```

Comment:
Same thing as above, if we're just going to re-throw we shouldn't catch at all

Reply:
Agreed, if we don't handle the error, there is no need to catch it.

Reply:
done. remove try..catch block.